package.json

```diff
@@ -10,17 +13,21 @@
     "rollup-plugin-svelte": "^5.0.3",
     "rollup-plugin-terser": "^4.0.4",
     "sirv-cli": "^0.4.4",
-    "svelte": "^3.0.0"
+    "svelte": "^3.0.0",
+    "svelte-loader": "^2.13.6"
   },
```

Comment:
We should also run `npm update` on these. E.g. svelte is at 3.9.2. :exclamation: